bobweb/bob/command_gpt.py

```diff
@@ -157,12 +156,7 @@ async def generate_and_format_result_text(update: Update) -> string:
 
     await send_bot_is_typing_status_update(update.effective_chat)
 
-    # For variety to user, instead of default model, use google's model (every other time)
-    # This assumes that google's model has similar capabilities as default model
-    if model.name == DEFAULT_MODEL.name and random.random() < 0.5:  # NOSONAR
-        model_name = 'gemini/gemini-2.5-flash'
-    else:
-        model_name = f"openai/{model.name}"
```

Comment:
I think this is a welcome change as user had no way to know what to think of the results as same message thread could be handled by random by either model and no one knew which answered. Not that there was any prominen usage after the last update.